eventmesh/constants.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = require('../common/constants');
+
+module.exports = Object.freeze({
+  LOCK_TYPE: {
+    WRITE: 'WRITE',
+    READ: 'READ'
+  },
+  LOCK_TTL: 5,
```

Comment:
What is typical read/write latency for etcd? We need to carefully set this ttl, it should be big enough to perform get, put and release operations.

Reply:
usually, it happens within seconds, so , I guess we are good there.

Reply:
Okay.